doc-tool/src/dotty/tools/dottydoc/staticsite/Site.scala

```diff
@@ -409,7 +409,7 @@ case class Site(
   }
 
   private def toSourceFile(f: JFile): SourceFile =
```

Comment:
Should probably be changed to take a `JPath` instead

Reply:
This will be for another PR.